dashboard-frontend/src/helpers/numbers.ts

```diff
@@ -0,0 +1,74 @@
+export const calculatePercentage = ({
```

Comment:
Great set of helpers, well tested! Will be useful.

Reply:
I'm glad that you like them 🙌